### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by SamuelTTW**
a folder called "img-desktop" is added which contains all the backgrounds of the desktop version, the desktop structure is added and cleaned in the file "index.html", the stylization of the desktop structure is added in "style.css"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,8 +322,8 @@ <h3>Skills</h3>
 
               <section class="formulary">
                 <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-
-                <form id="data" action="https://formspree.io/f/mvolyzzk" method="POST">
+                  <form class="form1" id="data" action="https://formspree.io/f/mvolyzzk" method="POST">
```
Comment (Kossi-stack) [inline index.html:325]: - [ ] Kindly fix the contact section of your design so it looks like [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- Your Design
Captureftp
- Figma Design
Capturefft

File: index.html [hunk 2]
```diff
@@ -204,10 +222,11 @@ <h3 class="cards-title">Multi-Post Stories Gain+Glory</h3>
             <section class="about-me">
 
               <div class="backaboutone"><img src="./img/background-1-about-me.svg" alt=""></div>
+              <div class="backaboutone-desktop"><img src="./img-desktop/about-me-1-desktop.svg" alt=""></div>
```
Comment (Kossi-stack) [inline index.html:225]: - [ ] Kindly fix the about me section of your design so it looks like [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- Your Design
Captureabtm
- Figma Design
Capturefabtm

File: index.html [hunk 3]
```diff
@@ -59,7 +77,7 @@ <h2>I`m a sofware developer</h2>
 
                 <section class="my-recent-work">
                   <h2>My Recent Works</h2>
```
Comment (Kossi-stack) [inline index.html:79]: - [ ] Kindly fix my works section of your design so it looks like [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- Your Design
Capturets
- Figma Design
Capturefts

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @SamuelTTW,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️
 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [ ] Kindly remove the live demo link in your pull request message and put it in the Readme
 

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by uwadonat — APPROVED** [review]
## Hi **@SamuelTTW,**

Thank you for implementing requests suggested by previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
appr2

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**